dev/TabView/TabView.idl

```diff
@@ -0,0 +1,74 @@
+﻿namespace MU_XC_NAMESPACE
+{
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+enum TabViewWidthMode
+{
+    SizeToContent = 0,
+    Fixed = 1,
+};
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+runtimeclass TabViewTabClosingEventArgs
+{
+    Object Item { get; };
+
+    Boolean Cancel { get; set; };
+}
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+[MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+[MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
+unsealed runtimeclass TabView : Windows.UI.Xaml.Controls.ListView
+{
+    TabView();
+
+    [MUX_DEFAULT_VALUE("winrt::TabViewWidthMode::SizeToContent")]
+    TabViewWidthMode TabWidthMode{ get; set; };
+
+    [MUX_DEFAULT_VALUE("true")]
+    Boolean CanCloseTabs{ get; set; };
+
+    Object LeftCustomContent{ get; set; };
+    Windows.UI.Xaml.DataTemplate LeftCustomContentTemplate{ get; set; };
+
+    Object RightCustomContent{ get; set; };
+    Windows.UI.Xaml.DataTemplate RightCustomContentTemplate{ get; set; };
+
+    event Windows.Foundation.TypedEventHandler<TabView, TabViewTabClosingEventArgs> TabClosing;
+
+    static Windows.UI.Xaml.DependencyProperty TabWidthModeProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty CanCloseTabsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty LeftCustomContentProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty LeftCustomContentTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty RightCustomContentProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty RightCustomContentTemplateProperty{ get; };
+}
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+[MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+[MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
+unsealed runtimeclass TabViewItem : Windows.UI.Xaml.Controls.ListViewItem
+{
+    TabViewItem();
+
+    String Header{ get; set; };
+
+    Windows.UI.Xaml.DataTemplate HeaderTemplate{ get; set; };
+
+    Windows.UI.Xaml.Controls.IconElement Icon{ get; set; };
+
+    [MUX_DEFAULT_VALUE("true")]
+    Boolean IsCloseable{ get; set; };
```

Comment:
>IsCloseable [](start = 12, length = 11)

I think WinRT would spell this "Closable" (such as IClosable)

Reply:
Hmm. I tried to figure out which is correct but it's not very clear at all. English is a horrible language... The names of all the properties are TBD anyway, though, so I'll have this debate with my PM later when we go through the spec!

Reply:
Yeah, this didn't come up in our review with the Toolkit.  Though based on existing docs they use `closeable` in the text but `closable` in the API name: E.g. https://docs.microsoft.com/en-us/windows/desktop/api/windows.foundation/nn-windows-foundation-iclosable and [search](https://www.bing.com/search?q=closable++Closeable+site%3adocs.microsoft.com&qs=n&sp=-1&pq=closable+closeable+site%3adocs.microsoft.c&sc=0-40&sk=&cvid=385DE1FAC3064C26A96DC6409B5AC341&first=1&FORM=PQRE)